src/StreetNameRegistry.Api.BackOffice.Handlers.Lambda/MessageHandler.cs

```diff
@@ -115,6 +115,18 @@ public async Task HandleMessage(object? messageData, MessageMetadata messageMeta
                         Provenance = request.ProvenanceData.ToProvenance()
                     }, cancellationToken);
                     break;
+
+                case SqsStreetNameCorrectRetirementRequest request:
```

Comment:
@ArneD do we want to add a default case statement where we throw an exception? Currently when we forget a mapping from an sqs request to a lambda request, the message is technically handled "correct" and thus disappears from the queue. Which means we lost a message.

Reply:
good idea, a `throw new NotImplementedException()` would be good => also in the other registries.